examples/withTempChannelsManager.js

```diff
@@ -23,7 +23,9 @@ client.on('ready', () => {
     childAutoDeleteIfParentGetsUnregistered: true,
     childAutoDeleteIfOwnerLeaves: false,
     childVoiceFormat: (str, count) => `Example #${count} | ${str}`,
-    childVoiceFormatRegex: /^Example #\d+ \|/
+    childVoiceFormatRegex: /^Example #\d+ \|/,
+    childMaxUsers: 0,
+    childBitrate: 96000
```

Comment:
By default, Discord sets the bitrate to 64Kbps.

```suggestion
    childBitrate: 64000
```

Reply:
Fixed! See 8928df1